sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-part-time-integration/e-cert-files/e-cert-file-record.ts

```diff
@@ -0,0 +1,193 @@
+    record.appendWithStartFiller(this.certNumber, 7, NUMBER_FILLER);
+    record.appendDate(this.educationalStartDate, DATE_FORMAT);
+    record.appendDate(this.educationalEndDate, DATE_FORMAT);
+    record.appendWithEndFiller(this.federalInstitutionCode, 4, SPACE_FILLER);
+    record.appendWithStartFiller(this.fieldOfStudy, 4, NUMBER_FILLER);
+    record.append(this.yearOfStudy.toString(), 1);
+    record.appendWithStartFiller(this.weeksOfStudy, 3, NUMBER_FILLER);
+    record.appendDate(this.documentProducedDate, DATE_FORMAT);
+    record.repeatAppend(SPACE_FILLER, 8); // TODO Cancel Date, E-cert cancellation date.
+    record.repeatAppend(SPACE_FILLER, 9); //CAG PD Amt, No longer used.
+    record.repeatAppend(SPACE_FILLER, 9); //CAG LI Amt, No longer used.
+    record.appendWithStartFiller(this.totalGrantAmount, 5, NUMBER_FILLER); //Sum of all CSGP grants + BC Provincial Part-time grants from this certificate.
+    record.appendWithStartFiller(this.totalCSGP_PTAmount, 5, NUMBER_FILLER); //Amount of Grant for Part-time Studies (CSGP-PT) at the study start.
+    record.repeatAppend(NUMBER_FILLER, 5); // CSGP NBD MI Amt, No longer used.
+    record.appendWithStartFiller(this.totalCSGP_PDAmount, 5, NUMBER_FILLER); //Amount of Grant for Students with Permanent Disabilities (CSGP-PD) at the study start.
+    record.appendWithStartFiller(this.totalCSGP_PTDEPAmount, 5, NUMBER_FILLER); //Amount Grant for Part-time Students with Dependants (CSGP-PTDEP) at the study start.
+    record.repeatAppend(NUMBER_FILLER, 5); // Amount of Grant for Services and Equipment for Students with Permanent Disabilities (CSGP-PDSE) at the study start, No longer used.
+    record.appendWithStartFiller(this.totalBCSGAmount, 5, NUMBER_FILLER); // BC Part-time grant amount 1
+    record.repeatAppend(NUMBER_FILLER, 5); // BC Part-time grant amount 2 - Reserved for future use
+    record.repeatAppend(SPACE_FILLER, 10); // Space Filler.
+    record.repeatAppend(SPACE_FILLER, 8); // CSGP MP Date, No longer used.
+    record.repeatAppend(SPACE_FILLER, 5); // CSGP MP PT Amt, No longer used.
+    record.repeatAppend(SPACE_FILLER, 5); // CSGP MP MI Amt, No longer used.
+    record.repeatAppend(SPACE_FILLER, 5); // CSGP MP PD Amt, No longer used.
+    record.repeatAppend(SPACE_FILLER, 5); // CSGP MP PTDEP Amt, No longer used.
+    record.repeatAppend(SPACE_FILLER, 5); // CSGP MP PDSE Amt, No longer used.
+    record.repeatAppend(SPACE_FILLER, 20); // Space Filler.
+    record.appendWithEndFiller(this.emailAddress, 75, SPACE_FILLER);
+    record.append("P"); // 'P' for part-time. Full time is done by another integration to another system.
+    record.appendDate(this.enrollmentConfirmationDate, DATE_FORMAT);
+    record.repeatAppend(SPACE_FILLER, 7); // EI Remittance Amt, optional, not provided.
+    record.appendWithStartFiller(50, 2, NUMBER_FILLER); // TODO Course Load Percentage, also hardcoded in assessment-gateway workflow.
```

Comment:
Maybe these constants can be defined at a higher level and the file is prepared to send it accordingly.
For instance, the ECertRecord can have the property and it could be hardcoded there.